src/import/loadConfigurationFromIgResource.ts

```diff
@@ -56,14 +58,16 @@ export function loadConfigurationFromIgResource(input: string): Configuration |
   });
   // Extract the configuration from the resource
   if (igResource && igResource.url && !multipleIgs) {
-    logger.info(`Extracting FSHOnly configuration from ${igPath}.`);
-    return {
+    const config = {
       canonical: igResource.url.replace(/\/ImplementationGuide.*/, ''),
-      FSHOnly: true,
-      dependencies: igResource.dependsOn?.filter(dep => dep.packageId && dep.version),
+      version: igResource.version,
       fhirVersion: igResource.fhirVersion ?? [],
-      version: igResource.version
+      dependencies: igResource.dependsOn?.filter(dep => dep.packageId && dep.version),
+      FSHOnly: true
     };
+    logger.info(`Extracting FSHOnly configuration from ${igPath}:`);
+    Object.entries(config).forEach(e => logger.info(`  ${e[0]}: ${JSON.stringify(e[1])}`));
```

Comment:
Do you think it would be worthwhile to format the JSON string that's logged here? With how it works now, I got a pretty dense block of text logged: 

![image](https://user-images.githubusercontent.com/30803904/96262489-d5677800-0f8f-11eb-9660-1a4a4d4f5e71.png)

I did try using `JSON.stringify(e[1], null, 2)`, and I'm not entirely sure it's better because the "info" boxes, but figured I'd ask your opinion.

![image](https://user-images.githubusercontent.com/30803904/96262693-25463f00-0f90-11eb-8693-baaadf38f749.png)

Reply:
I also tried `JSON.stringify` when I was developing this and decided I liked it better _without_ it.  It just takes up a lot of space with it and looks kind of funky with the labels.  In reality, these log messages are mainly helpful for debugging, so they don't need to be super readable every time.  And also, usually people will have at most one IG dependency (at least now that seems to be the case) -- so the block won't be as large as yours above.

All that said, I'm willing to go the other way if you all think it's better to format it.

Reply:
Although, if we're going to use JSON stringify with formatting, then maybe it would be better not to use a separate log message per property but to put the whole thing in a single log message (e.g., "Extracting FSHOnly configuration..." with the formatted JSON appended to the same message).

Reply:
Yeah I agree that the "info" labels make it look kind of funky, and given your reasoning, I'm good leaving it as it is!